.circleci/config.yml

```diff
@@ -103,29 +103,6 @@ jobs:
           name: Hardhat Test
           command: yarn test:integration:polygon
 
-  test_integration_optimism:
```

Comment:
We're removing optimism integration tests?

Reply:
Oh no. this is a mistake. Great catch.
I was wondering why this pipeline wasn't triggered in the last commit🤦‍♂️

Reply:
Added it back in.
FYI: This happened because when I wanted to reset this file to the master version I used my fork master (which doesn't have this test yet) instead of the upstream/master.